api/pkg/service/resource/resource_http_test.go

```diff
@@ -42,7 +42,24 @@ func TestQuery_Http(t *testing.T) {
 	tc := testutils.Setup(t)
 	testutils.LoadFixtures(t, tc.FixturePath())
 
-	QueryChecker(tc).Test(t, http.MethodGet, "/query?name=build&kind=pipeline&limit=1").Check().
+	QueryChecker(tc).Test(t, http.MethodGet, "/query?name=build&kinds=pipeline&limit=1").Check().
```

Comment:
could you please add more tests?
- passing only tags
- pass only kinds
- try to cover each case